generated/nirfmxinstr/nirfmxinstr.proto

```diff
@@ -1001,7 +1148,10 @@ message SelectActiveExternalAttenuationTableResponse {
 message SelfCalibrateRequest {
   nidevice_grpc.Session instrument = 1;
   string selector_string = 2;
-  int32 steps_to_omit = 3;
+  oneof steps_to_omit_enum {
+    SelfCalStep steps_to_omit = 3;
+    int32 steps_to_omit_raw = 4;
```

Comment:
Question: Do you have any background on why some self-cal steps are int 64 and some int 32? Based on the functions I see both flavor APIs

Reply:
It's just RFSA/G that have 64-bit self cal steps, right? And RFmxInstr is 32-bit? I assume that it's because RFSA/G conservatively reserved a ton of bits for self cal steps and RFmx decided not to. Not sure why that was but `protobuf`-compatibility turns out to be a good example of a reason to stick with 32-bits for enum-like-encodings if you can get away with it: it represents enums as int32s.